llama_stack/templates/huggingface/huggingface.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
I am not convinced we should have a `huggingface` distribution, just to house post training. Is the addition to the `ollama` distribution not sufficient? What other distributions do you think this provider should come "built-in" with?

Reply:
I think either is fine. Having a post_training distro that is lightweight to point users to in the docs might be beneficial though to hopefully get some community adoption of the post_training API.

Reply:
changed it so that this provider is in the experimental_post_training distro